src/openrct2/ride/vehicle.h

```diff
@@ -77,8 +77,11 @@ typedef struct rct_ride_entry_vehicle {
 	uint8 draw_order;
 	uint8 special_frames;			// 0x60 , 0x7A
 	sint8* peep_loading_positions;	// 0x61 , 0x7B
+#ifdef NO_RCT2
+	uint16 peep_loading_positions_count;
+#endif
 } rct_ride_entry_vehicle;
-#ifdef PLATFORM_32BIT
+#if defined(PLATFORM_32BIT) && !defined(NO_RCT2)
```

Comment:
Could you also check for  `#if defined(PLATFORM_32BIT) && defined(NO_RCT2)` with the new struct size?